src/config.ts

```diff
@@ -29,6 +30,7 @@ const configSchema = z.object({
   runScriptTimeoutMilliseconds: z.number().positive(),
   rawMemoryLimit: z.string().optional(),
   rawCpuLimit: z.string().optional(),
+  filesDir: z.string(),
```

Comment:
Given that the env var is optional, shall we make this one optional as well? So that if there is no filesDir at all, we just don't mount the volume when we spin up the docker container